src/page-multi-facility/AddFacilityModal/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import Colors from "../../design-system/Colors";
+import InputButton from "../../design-system/InputButton";
+import Modal from "../../design-system/Modal";
+import ModalSteps from "../../design-system/ModalSteps";
+import { EpidemicModelProvider } from "../../impact-dashboard/EpidemicModelContext";
+import LocaleInformationPage from "./LocaleInformationPage";
+
+const AddFacilityModalContainer = styled.div``;
+
+const ModalFooter = styled.div`
+  border-top: 0.5px solid ${Colors.darkGray};
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  padding: 1em 0;
+  width: 100%;
+`;
+
+const AddFacilityModal: React.FC = () => {
+  const [activeStep, setActiveStep] = useState(1);
+  const numSteps = 3;
+
+  const handleButtonClick = () => {
+    if (activeStep < numSteps) {
+      setActiveStep(activeStep + 1);
+    }
+  };
+
+  return (
+    <EpidemicModelProvider>
```

Comment:
I think probably you are going to want to wrap the entire multi-facility page in this Provider so that everything on it shares the same context data? I am not entirely sure how the context will evolve to accommodate multi-facility data but I think this much is a safe assumption